app/src/main/java/swati4star/createpdf/fragment/HomeFragment.java

```diff
@@ -283,8 +330,79 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
             mMorphButtonUtility.morphToSquare(mCreatePdf, mMorphButtonUtility.integer());
             mImageCounter++;
             next();
+        } else if (requestCode == INTENT_REQUEST_APPLY_FILTER) {
+            switch (resultCode) {
+                case Activity.RESULT_OK:
+                    try {
+                        mImagesUri.clear();
+                        mTempUris.clear();
+                        mFilterUris = data.getStringArrayListExtra("result");
+                        int size = mFilterUris.size() - 1;
+                        for (int k = 0; k <= size; k++) {
+                            mTempUris.add(mFilterUris.get(k));
+                        }
+                    } catch (Exception e) {
+                        e.printStackTrace();
+                    }
+                    mImageCounter++;
+            }
         }
     }
+    /**
```

Comment:
I have added the morph button related functions in https://github.com/Swati4star/Images-to-PDF/blob/master/app/src/main/java/swati4star/createpdf/util/MorphButtonUtility.java file. There is no need to add these here.